scripts/create-github-release.js

```diff
@@ -131,20 +132,19 @@ async function getPrDescriptionFromCommitMessage(commitMessage) {
     `Identified the changeset Pull request - https://github.com/backstage/backstage/pull/${prNumber}`,
   );
 
-  const prData = await octokit.pulls.get({
+  const { data } = await octokit.pulls.get({
     owner: GH_OWNER,
     repo: GH_REPO,
     pull_number: prNumber,
   });
 
-  return prData.data.body;
-}
+  // Use the PR description to prepare for the release description
+  const isChangesetRelease = commitMessage.includes(CHANGESET_RELEASE_BRANCH);
+  if (isChangesetRelease) {
+    return data.body.split('\n').slice(3).join('\n');
+  }
 
-// Use the PR description to prepare for the release description
-async function prepareReleaseDescription(prDescription) {
-  // TODO: Refine prDescription to remove the lines containing "Update Dependencies"
-  // Remove everything in the beginning until changelogs.
```

Comment:
Is it done or we don't want to do it? 😄

Reply:
Don't want it :grin:

Don't think it takes too much space and it can be good to keep track of as well. Doesn't deserve a TODO either way I think :p

Reply:
Cool!